### PR TITLE
migrate examples/javascript/predict-tense.js to examples/typescript/predict-tense.ts

### PR DESCRIPTION
**Pull request by luisgregson**
Creates an equivalent typescript example for the existing javascript predict-tense example

## Description
Copied the examples/javascript/predict-tense.js file to examples/typescript/predict-tense.ts and fixed import format.

## Motivation and Context
This is part of a migration to typescript covered by issue [#551](https://github.com/BrainJS/brain.js/issues/551)

## How Has This Been Tested?
Ran the linter and compiled the file with the tsc binary in node_modules

## Screenshots (if appropriate):
N/A

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Author's Checklist:

- [x] My code focuses on the main motivation and avoids scope creep.
- [x] My code passes current tests and adds new tests where possible.
- [x] My code is [SOLID](https://en.wikipedia.org/wiki/SOLID_(object-oriented_design)) and [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).
- [x] I have updated the documentation as needed.

## Reviewer's Checklist:
- [ ] I kept my comments to the author positive, specific, and productive.
- [ ] I tested the code and didn't find any new problems.
- [ ] I think the motivation is good for the project.
- [ ] I think the code works to satisfies the motivation.

### CODE REVIEW

File: examples/typescript/predict-tense.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  { input: 'Am I wrong?', output: 'present' },
+  { input: 'Right this way.', output: 'present' },
+];
+
+// the thing we would test
+const test = 'Will you be my friend?';
+
+const network = new brain.recurrent.LSTM();
+network.train(data, config);
+const output = network.run(test);
```
Comment (mubaidr) [inline examples/typescript/predict-tense.ts:52]: we can cast result to `string`

**Review by mubaidr — CHANGES_REQUESTED** [review]
Cast output to proper type i.e. `string` in our case

File: examples/typescript/predict-tense.ts (hunk 1 above)
Comment (luisgregson) [reply examples/typescript/predict-tense.ts:52]: I'm not sure i understand. Can you give me an example of how you would like it to look?

The template literal automatically casts `output` to string, and i think it makes sense for the `output` variable to infer its type from the  `network.run` method?

I could give `output` the type of string (`const output: String = network.run(test);`) in order to cause an error if `run` is typed to return something else?

Comment (luisgregson) [reply examples/typescript/predict-tense.ts:52]: or i could make sure it is always a string regardless of what `network.run(test)` returns by doing this?
```
const output = String(network.run(test));
```

**Comment by luisgregson** [comment]
> Cast output to proper type i.e. `string` in our case

I've added a question to the comment, as i wasn't sure what the change should be

File: examples/typescript/predict-tense.ts (hunk 1 above)
Comment (mubaidr) [reply examples/typescript/predict-tense.ts:52]: Good point! The result will be typed once we have `rnn` network typed.

**Pull request merged**